enterprise/frontend/src/embedding-sdk/cli/steps/setup-metabase-instance.ts

```diff
@@ -141,37 +138,6 @@ export const setupMetabaseInstance: CliStepMethod = async state => {
 
     const cookie = res.headers.get("set-cookie") ?? "";
 
-    spinner.succeed();
-    spinner.start("Enabling embedding features...");
-
-    res = await fetch(`${state.instanceUrl}/api/setting`, {
```

Comment:
We move the embedding settings to its own step, so we can call this once the license has been activated.